azure-pipelines.yml

```diff
@@ -1,3 +1,25 @@
+trigger:
+  # Only run latest commit for branches:
+  batch: true
```

Comment:
Do we want to batch or use `autoCancel`? I think we currently do `autoCancel` to only test the latest commit on PRs